### DISCUSSION TITLE
UI discrepancy in macOS Monterey

### DISCUSSION THREAD

**Discussion by andydoan** [dooit-org/dooit#100] [General]
<img width="294" alt="Screen Shot 2023-03-13 at 3 03 15 PM" src="https://user-images.githubusercontent.com/3294357/224805862-79d3bb87-c658-42cd-b177-8e6ba11e4477.png">

With a fresh install via `brew install dooit`, the dooit UI looks different than the [examples](https://user-images.githubusercontent.com/97718086/221467485-fae198f7-51b1-4a71-91d9-88b51897aeeb.png) provided in the repo readme.

Is this perhaps a dooit configuration issue or an issue with my terminal/macOS settings?

**Comment by kraanzu** [comment]
Hey there @andydoan !

Yes, those are custom configs! In the previous version of dooit I tried to make it look great from start but I got many complaints for fonts and such so I kept the default config simple as possible!

You can see how to configure the app [here](https://github.com/kraanzu/dooit/wiki/Configuration)!

In the examples, I used my custom configs from [here](https://github.com/kraanzu/.dotfiles/tree/main/dooit/.config/dooit)
My configs are a bit messy! Need to clean them up. But all stuff is pretty straight forward and easy and If you want some custom stuff you can use python scripts :D

**Reply by andydoan** [reply]
Ah, I must have glossed over the wiki and went straight to using dooit! Sorry to be another complainer of fonts, but do you happen to know what fonts your custom configs are using? I love the icons but it looks like I'm missing a font(s) after cloning your dotfiles :sweat_smile: 

Thanks for your service to TUIs!

**Reply by kraanzu** [reply]
Oh I am using [Nerd Fonts](https://www.nerdfonts.com/)

> Thanks for your service to TUIs!

I'm glad to be of service :smile:
